src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import './vars';
+@import './extends';
+@import './mixins';
+@import './keyframes';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @extend %stopwatch-flex-property;
```

Comment:
You are using `@extend` to inherit from `%stopwatch-flex-property`. While this is not wrong, it's generally better to use `@include` with a mixin for this kind of use case. Mixins are more flexible than extends, as they can accept arguments and are not limited by the CSS inheritance rules. Consider changing `%stopwatch-flex-property` to a mixin.